benchmarks/src/main/kotlin/com/google/samples/apps/nowinandroid/foryou/ForYouActions.kt

```diff
@@ -41,6 +44,12 @@ fun MacrobenchmarkScope.forYouWaitForContent() {
  */
 fun MacrobenchmarkScope.forYouSelectTopics(recheckTopicsIfChecked: Boolean = false) {
     val topics = device.findObject(By.res("forYou:topicSelection"))
+    val withChildren = topics.childCount != 0
+    if (!withChildren) {
+        // TODO: Ensure ForYou has topics.
+        Log.w(TAG, "no topics found, can't scroll for baseline profile generation.")
```

Comment:
I think we should keep failing the generator if there's no topic 🤔 . I think this happens when something is loading longer than this test anticipates. Or am I missing something?

Reply:
Failing now.